refactor(book): split pages and update RPC docs (#487)
botanix-labs/Macbeth: book/run/rpc-server-docker.md

>
> Reth requires Docker Engine version 20.10.10 or higher due to [missing support](https://docs.docker.com/engine/release-notes/20.10/#201010) for the `clone3` syscall in previous versions.

## Prerequisites

> In order to be able to use the instructions below, you need to run a mutinynet bitcoind instance. This node needs to be completely synced to the tip before you can continue with the Docker compose instructions which follow below. Instructions on how to run your own instance can be found on these links:

Reviewer: ```
To use the instructions below, you’ll need to run mutinynet signet node. Ensure that this node is fully synced to the tip before proceeding with the Docker Compose instructions provided afterward. You can find instructions for running your own node at the following links:
```

Re-word

 - [https://github.com/benthecarman/bitcoin/releases](https://github.com/benthecarman/bitcoin/releases)
 - [https://github.com/MutinyWallet/mutiny-net](https://github.com/MutinyWallet/mutiny-net)

 > **Note**
 >

>  > Mutinynet is a fork of bitcoin core that allows for fast 30 second signet blocks.  This makes testing quite fast as coins and lightning channels are available almost instantly. There is a whole suite of tools available for MutinyNet, including [coin faucet](https://faucet.mutinynet.com) and [block explorer](https://mutinynet.com).

Reviewer: ```suggestion
 > Mutinynet is a fork of bitcoin core that is configured for 30 second blocks. This allows our team to test more rapidly. There is a whole suite of tools available for MutinyNet, including [coin faucet](https://faucet.mutinynet.com) and [block explorer](https://mutinynet.com).
```
